src/senaite/core/content/samplematrix.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of SENAITE.CORE.
+#
+# SENAITE.CORE is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright 2018-2024 by it's authors.
+# Some rights reserved, see README and LICENSE.
+
+from bika.lims.interfaces import IDeactivable
+from plone.supermodel import model
+from senaite.core.catalog import SETUP_CATALOG
+from senaite.core.content.base import Container
+from senaite.core.interfaces import ISampleMatrix
+from zope import schema
+from zope.interface import implementer
+
+
+class ISampleMatrixSchema(model.Schema):
+    """Schema interface
+    """
+
+    title = schema.TextLine(
+        title=u"Title",
+        required=False,
+    )
+
+    description = schema.Text(
+        title=u"Description",
+        required=False,
+    )
```

Comment:
Would you mind to add here our message factories with the `i18n` IDs?
Shall we use `Name` instead of `Title` in general for our Setup items?

Reply:
Done with https://github.com/senaite/senaite.core/pull/2501/commits/f05d2242c5639222fadd1b51c46dd3c7fe375b1f . I've updated the rest of setup DX types accordingly with https://github.com/senaite/senaite.core/pull/2501/commits/5daf1841bd8bc922f4bffdb0fc8b81c7d5477ce8